erpnext/hr/doctype/job_offer/job_offer.js

```diff
@@ -12,6 +12,38 @@ frappe.ui.form.on("Job Offer", {
 		});
 	},
 
+	salary_structure: function(frm) {
+		if (frm.doc.salary_structure) {
+			frappe.model.with_doc("Salary Structure", frm.doc.salary_structure, function() {
+				var tabletransfer = frappe.model.get_doc("Salary Structure", frm.doc.salary_structure);
+				let earning = [];
+				$.each(tabletransfer.earnings, function(index, row){
+					let d = {};
+					d.salary_component = row.salary_component;
+					d.abbr = row.abbr;
+					d.formula = row.formula;
```

Comment:
You cannot show formula here and send it to the job applicant.

You have to execute the salary structure for a specific month and show the calculated value.

Check "Show Preview" code in salary structure for reference.